packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -50,16 +52,30 @@ export interface _CaptionsBannerProps {
    * @defaultValue 'default'
    */
   formFactor?: 'default' | 'compact';
+  captionsOptions?: {
+    height: 'full' | 'default';
+  };
 }
 
+const SCROLL_OFF_SET_ALLOWANCE = 20;
```

Comment:
What is this offset for?

Reply:
Interesting, what I can still pass the build, LOL, lemme fix this